zkevm-circuits/src/evm_circuit.rs

```diff
@@ -186,8 +186,8 @@ pub mod test {
         Word::from_big_endian(&rand_bytes_array::<32>())
     }
 
-    pub(crate) fn rand_fp() -> Fp {
-        Fp::rand()
+    pub(crate) fn rand_scalar() -> Fr {
+        Fr::rand()
```

Comment:
Is this fn used anywhere?? 

If not, we could remove it directly. Since is the same to do `Fr::rand()` than `rand_scalar()`